### ISSUE TITLE
Missing imports

### ISSUE THREAD

**Issue by wasyster** [postsharp/Metalama.Samples#7]
Hi!

I am creating a custom log attribute, and I run into an error when tried to convert parameter object to json. I have the following lines in my code:

```
    protected InterpolatedStringBuilder BuildInterpolatedString()
    {
        var stringBuilder = new InterpolatedStringBuilder();

        stringBuilder.AddText(meta.Target.Method.Name);

        stringBuilder.AddText("(");

        var i = meta.CompileTime(0);

        foreach (var prop in meta.Target.Parameters)
        {
            var comma = i > 0 ? ", " : "";

            if (prop.RefKind == RefKind.Out)
            {
                stringBuilder.AddText($"{comma}{prop.Name} = <out> ");
            }
            else
            {
                
                stringBuilder.AddText(prop.Name);
                stringBuilder.AddText(" : ");
                var json = System.Text.Json.JsonSerializer.Serialize(prop.Value);
                stringBuilder.AddText(json);
                //stringBuilder.AddExpression(prop.Value);
            }

            i++;
        }

        stringBuilder.AddText(")");

        return stringBuilder;
    }
```
On run I get the following error: **CS0234: The type or namespace name 'Json' does not exist in the namespace 'System.Text' (are you missing an assembly reference?)**

When I inspect the converted code I found out that the System.Text.Json not got imported it is missing!
The code will not compile even if I write the full namespace like:
var json = **System.Text.Json.JsonSerializer.Serialize**(prop.Value);

**Comment by gfraiteur** [comment]
The root cause if that the `JsonSerializer` class cannot be used in compile-time code.
The root cause is masked by a cryptic error message, and we will fix this situation.

**Comment by addabis** [comment]
Duplicate of postsharp/Metalama#13

**Comment by gfraiteur** [comment]
This has been fixed in 0.5.12.
